content/en/blog/_posts/2023-04-11-topology-spread-features.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: blog
+title: "TBD" // TODO: have a cool title.
```

Comment:
@denkensk @kerthcet any ideas? 🙂

Reply:
How about <Introducing the advanced pod topology spread policy for Kubernetes>?

Reply:
or `More fine-gained pod topology spread policies in Kubernetes scheduling`.

Reply:
I like the word "fine-gained". 
Also, I'd like to mention that these features are coming to beta in v1.27 like [many other blog posts did](https://kubernetes.io/blog/page/3/).

So... like `Kubernetes 1.27: More fine-gained pod topology spread policies reached beta`?

Reply:
`Kubernetes 1.27: More fine-grained pod topology spread policies reached beta` 
sgtm.